.github/workflows/autobuild.yml

```diff
@@ -159,13 +159,13 @@ jobs:
 
           - config_name:            Windows (artifact+codeQL)
             target_os:              windows
-            building_on_os:         windows-2019
+            building_on_os:         windows-latest
```

Comment:
Is there a pinnable version?

Reply:
Yes. windows-2022 probably

Reply:
I'd prefer that rather than something that could change.